src/components/voteLogLegend.js

```diff
@@ -199,42 +199,30 @@ const VoteLogLegend = ({
           </>
         )}
       </span>
-      <span css={cssLegendWrap({ missing, type })}>
-        <img
-          src={cross}
-          style={{
-            margin: "0",
-          }}
-          width="8"
-          height="8"
-          alt={
-            missing !== undefined
-              ? type === "group"
-                ? ""
-                : "ไม่ลงมติ"
-              : "ไม่เข้าประชุม"
-          }
-        />{" "}
-        {missing !== undefined ? (
-          type === "group" ? (
-            <>
-              <b style={{ margin: "0 0.3rem" }}></b> {special}%
-            </>
-          ) : type === "popup" ? (
-            <>
-              <b style={{ margin: "0 1rem" }}>ไม่ลงมติ</b> {special}
-            </>
-          ) : (
+      {missing === undefined && special !== undefined && (
+        <span css={cssLegendWrap({ missing, type })}>
+          <img
+            src={cross}
+            style={{
+              margin: "0",
+            }}
+            width="8"
+            height="8"
+            alt={
+              missing !== undefined
+                ? type === "group"
+                  ? ""
+                  : "ไม่ลงมติ"
+                : "ไม่เข้าประชุม"
+            }
+          />{" "}
+          {missing === undefined && (
```

Comment:
Will this condition will always be `true` since it's under `missing === undefined && special !== undefined` condition in line 202?